website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -48,7 +308,7 @@ Generates a new team token and overrides existing token if one exists.
 | :----- | :----------------------------------- |
 | POST   | /teams/:team_id/authentication-token |
```

Comment:
If the sample payload below ends in `s` then should this also, @mkam  ?

<img width="950" alt="image" src="https://github.com/user-attachments/assets/edc4f41d-d675-4443-b2e1-f0f74483ee3f" />

Reply:
Please correct me if I'm misunderstanding, but I believe that the comment that you're referring to meant to say `authentication-token`, singular, rather than `authentication-tokens`, plural. This being said, I've updated @mkam's comment and will leave this as-is since it's the legacy API!